src/dbnode/storage/bootstrap/bootstrapper/fs/source.go

```diff
@@ -1007,7 +1010,7 @@ func (s *fileSystemSource) bootstrapFromIndexPersistedBlocks(
 			s.log.Error("unable to read segments from index fileset",
 				zap.Stringer("namespace", ns.ID()),
 				zap.Error(err),
-				zap.Time("blockStart", indexBlockStart),
+				zap.Time("blockStart", indexBlockStart.ToTime()),
```

Comment:
Let's make sure we only do these conversions in error cases

Reply:
Did a sweep; these conversions happen largely in errors/invariant violation logging, but are also called as part of the trace path for some large granularity functions like top level `Query` calls, where the overhead will be negligible (makes some sense since we wouldn't have been logging the hotpath anyway)